server.js

```diff
@@ -1,248 +1,229 @@
 const inquirer = require('inquirer');
 const mysql = require('mysql');
 const cTable = require('console.table');
+let programIsRunning = true;
+
+const actions = {
+  "View all departments": viewAllDepartments,
+  "View all roles": viewAllRoles,
+  "View all employees": viewAllEmployees,
+  "Add a department": addADepartment,
+  "Add a role": addARole,
+  "Add an employee": addAnEmployee,
+  "Update employee roles": updateEmployeeRoles,
+  "Exit program": exitProgram,
+};
 
 const connection = mysql.createConnection({
-    host: 'localhost',
-    port: 3306,
-    user: 'root',
-    password: '',
-    database: 'employeeTrackerDB',
+  host: 'localhost',
+  port: 3306,
+  user: 'root',
+  password: '',
+  database: 'employeeTrackerDB',
 });
 
 connection.connect((err) => {
-    if (err) throw err;
-    console.log(`Connected as id ${connection.threadId} \n`);
-    startApp();
+  if (err) throw err;
+  console.log(`Connected as id ${connection.threadId} \n`);
+  startApp();
 });
 
-startApp = () => {
-    inquirer.prompt([
-        {
-            name: 'initialInquiry',
-            type: 'rawlist',
-            message: 'Welcome to the employee management program. What would you like to do?',
-            choices: ['View all departments', 'View all roles', 'View all employees', 'Add a department', 'Add a role', 'Add an employee', 'Update employee roles', 'Exit program']
-        }
-    ]).then((response) => {
-        switch (response.initialInquiry) {
-            case 'View all departments':
-                viewAllDepartments();    
-                break;
-            case 'View all roles':
-                viewAllRoles();
-                break;
-            case 'View all employees':
-                viewAllEmployees();
-                break;
-            case 'Add a department':
-                addADepartment();
-            break;
-            case 'Add a role':
-                addARole();
-            break;
-            case 'Add an employee':
-                addAnEmployee();
-            break;
-            case 'Update employee roles':
-                updateEmployeeRole();
-            break;
-            case 'Exit program':
-                connection.end();
-                console.log('\n You have exited the employee management program. Thanks for using! \n');
-                return;
-            default:
-                break;
-        }
-    })
+function startApp() {
+  inquirer.prompt([
+    {
+      name: 'initialInquiry',
+      type: 'rawlist',
+      message: 'Welcome to the employee management program. What would you like to do?',
+      choices: Object.keys(actions)
+    }
+  ]).then((response) => {
+    const actionToExecute = actions[actionName];
+    if (actionToExecute) {
+      actionToExecute();
+    }
+
+    if (!programIsRunning) {
```

Comment:
@samrogers15 I'm unable to run your repo locally because I don't have the SQL stuff set up so I'm not sure if this is necessary, but I saw your exit functionality had a return statement so I thought there might be a need to maintain that statement. If it's not necessary, you could remove this condition and the usage of the `programIsRunning` variable everywhere.